reflex/app.py

```diff
@@ -130,6 +131,12 @@ class App(Base):
     # Components to add to the head of every page.
     head_components: List[Component] = []
 
+    # The language to add to the html root tag of every page.
+    html_lang: Optional[Union[Var[str], str]] = None
+
+    # Attributes to add to the html root tag of every page.
+    html_custom_attrs: Optional[Dict[str, Union[Var, str]]] = None
```

Comment:
For now, accessing state Var won't work in the _document.js file; so i think we should just accept `str` type instead of `Union[Var[str], str]`

Reply:
Fixed it for `html_lang`. How should we handle the `custom_attrs`? This attribute is inherited from the `Component` class.

Reply:
i think just accept it as a `str` in the App, and then you can convert it to a Var in the compile function, like you're already doing.

the main problem is that at the time `_document.js` is rendered by Next, the state hasn't been set up yet. So Vars in general are fine, but State Vars are not.

Reply:
Sounds logical.
But due to possible nested dicts this wouldn't be so easy to implement.

Shouldn't it be possible to dirty fix `custom_attrs` with a `# type: ignore`?
In the outer layer (app) we accept `Dict[str, str]` which can be "casted" to Html's `Dict[str, Union[Var, str]]` with the type ignoration. I want to achieve, that we can fully stick to the inherited `custom_attrs`.

Reply:
i dont see the need for the `# type: ignore`. When i change `html_lang` to `Optional[Dict[str, str]]` i don't get any type checking errors from pyright.

i would assume that `Dict[str, str]` is "compatible" with `Dict[str, Union[Var, str]]`

Reply:
Thanks for the detailed review. Is fixed.